### PR TITLE
Fix the link to developer policies in footer

### LINKED ISSUE #10223: Link to archived "Developer Policies" should point to updated content.
### Describe the problem and steps to reproduce it:
In the sitewide footer, we have a link to "Developer Policies" that points to this [archived page](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/AMO/Policy) - guessing there's an updated policy location to link to?

### What happened?
Link went to archived page.

### What did you expect to happen?
Link goes to current page :)

### Anything else we should know?
Screen Shot 2021-03-16 at 10 38 15 AM

### PR DESCRIPTION
**Pull request by championshuttler**
Fixes #10223

### CODE REVIEW

**Comment by championshuttler** [comment]
Not sure but i think this would be the correct link

File: src/amo/components/Footer/index.js [hunk 1]
```diff
@@ -76,7 +76,7 @@ export class FooterBase extends React.Component {
                 </Link>
               </li>
               <li>
-                <a href="https://developer.mozilla.org/docs/Mozilla/Add-ons/AMO/Policy">
+                <a href="https://extensionworkshop.com/documentation/publish/add-on-policies/">
```
Comment (willdurand) [inline src/amo/components/Footer/index.js:79]: We could probably use the `extensionWorkshopUrl` config parameter here (for consistency).

Comment (championshuttler) [reply src/amo/components/Footer/index.js:79]: good idea, is this the correct link though Will, because content is little diff on archive and this page?

Comment (willdurand) [reply src/amo/components/Footer/index.js:79]: I don't know, maybe @caitmuenster would know?

**Comment by championshuttler** [comment]
oops, clicked by mistake on re-review

File: src/amo/components/Footer/index.js (hunk 1 above)
Comment (caitmuenster) [reply src/amo/components/Footer/index.js]: That's the correct link! 👍

**Comment by caitmuenster** [comment]
@championshuttler Can you add UTM params to this? Should be: 

?utm_source=addons.mozilla.org&utm_medium=photon-footer

**Comment by championshuttler** [comment]
Sure thing @caitmuenster, just want to confirm 'photon-footer' should we replace all the existing params too?

**Comment by willdurand** [comment]
> Sure thing @caitmuenster, just want to confirm 'photon-footer' should we replace all the existing params too?

We don't have params right now?

**Comment by championshuttler** [comment]
> > Sure thing @caitmuenster, just want to confirm 'photon-footer' should we replace all the existing params too?
> 
> We don't have params right now?

Sorry I meant `utm_medium` because for now for others i guess there is no `utm_medium` atm

**Comment by caitmuenster** [comment]
Yeah, utm_medium=photon-footer is making an appearance courtesy of [issue #16753](https://github.com/mozilla/addons-server/issues/16753). ;)

File: bin/loading-page.html [hunk 2]
```diff
@@ -73,7 +73,7 @@ <h4 class="Footer-links-header"><a href="/en-GB/firefox/"><!-- react-text: 236 -
                 </a>
               </li>
               <li>
-                <a href="https://developer.mozilla.org/docs/Mozilla/Add-ons/AMO/Policy">Developer Policies
+                <a href="https://extensionworkshop.com/documentation/publish/add-on-policies/">Developer Policies
```
Comment (willdurand) [inline bin/loading-page.html:76]: No need to add the query params here.

Comment (championshuttler) [reply bin/loading-page.html:76]: Yeah I did not, though in below lines they are added, in browser links

**Review by willdurand — APPROVED** [review]

**Pull request merged**